<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtConfig</class>
 <widget class="QMainWindow" name="QtConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>554</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CustomBadges</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>531</width>
      <height>381</height>
     </rect>
    </property>
    <widget class="QCheckBox" name="overwolfCheckBox">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>50</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Overwolf</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>:/badge/icons/overwolf.png</normaloff>:/badge/icons/overwolf.png</iconset>
     </property>
    </widget>
    <widget class="QTreeWidget" name="badgeList">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>120</y>
       <width>431</width>
       <height>241</height>
      </rect>
     </property>
     <property name="dragDropOverwriteMode">
      <bool>false</bool>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::DragOnly</enum>
     </property>
     <property name="defaultDropAction">
      <enum>Qt::IgnoreAction</enum>
     </property>
     <property name="indentation">
      <number>0</number>
     </property>
     <column>
      <property name="text">
       <string>Badges</string>
      </property>
     </column>
    </widget>
    <widget class="QGroupBox" name="badge1Box">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>101</width>
       <height>80</height>
      </rect>
     </property>
     <property name="title">
      <string>Badge 1</string>
     </property>
     <widget class="ClickableLabel" name="badge1Label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>12</y>
        <width>64</width>
        <height>64</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="badge3Box">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>20</y>
       <width>101</width>
       <height>80</height>
      </rect>
     </property>
     <property name="title">
      <string>Badge 3</string>
     </property>
     <widget class="ClickableLabel" name="badge3Label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>12</y>
        <width>64</width>
        <height>64</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="badge2Box">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>20</y>
       <width>101</width>
       <height>80</height>
      </rect>
     </property>
     <property name="title">
      <string>Badge 2</string>
     </property>
     <widget class="ClickableLabel" name="badge2Label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>12</y>
        <width>64</width>
        <height>64</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QToolButton" name="manualBadgeOverride">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>80</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="acceptButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>390</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Annehmen</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rejectButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>390</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Abbrechen</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ClickableLabel</class>
   <extends>QLabel</extends>
   <header>ClickableLabel.h</header>
   <slots>
    <signal>clicked()</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>acceptButton</sender>
   <signal>clicked()</signal>
   <receiver>QtConfig</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>99</x>
     <y>331</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>187</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rejectButton</sender>
   <signal>clicked()</signal>
   <receiver>QtConfig</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>280</x>
     <y>331</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>187</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>acceptButton</sender>
   <signal>clicked()</signal>
   <receiver>QtConfig</receiver>
   <slot>saveToConfig()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>127</x>
     <y>333</y>
    </hint>
    <hint type="destinationlabel">
     <x>276</x>
     <y>230</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>badge1Label</sender>
   <signal>clicked()</signal>
   <receiver>QtConfig</receiver>
   <slot>deleteBadge1()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>121</x>
     <y>73</y>
    </hint>
    <hint type="destinationlabel">
     <x>276</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>badge2Label</sender>
   <signal>clicked()</signal>
   <receiver>QtConfig</receiver>
   <slot>deleteBadge2()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>231</x>
     <y>73</y>
    </hint>
    <hint type="destinationlabel">
     <x>276</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>badge3Label</sender>
   <signal>clicked()</signal>
   <receiver>QtConfig</receiver>
   <slot>deleteBadge3()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>341</x>
     <y>73</y>
    </hint>
    <hint type="destinationlabel">
     <x>276</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>manualBadgeOverride</sender>
   <signal>clicked()</signal>
   <receiver>QtConfig</receiver>
   <slot>openManualBadges()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>460</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>276</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>badgeList</sender>
   <signal>itemDoubleClicked(QTreeWidgetItem*,int)</signal>
   <receiver>QtConfig</receiver>
   <slot>addBadge(QTreeWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>275</x>
     <y>250</y>
    </hint>
    <hint type="destinationlabel">
     <x>276</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>badgeList</sender>
   <signal>itemDoubleClicked(QTreeWidgetItem*,int)</signal>
   <receiver>QtConfig</receiver>
   <slot>updateBoxes()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>275</x>
     <y>250</y>
    </hint>
    <hint type="destinationlabel">
     <x>276</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>badge1Label</sender>
   <signal>clicked()</signal>
   <receiver>QtConfig</receiver>
   <slot>updateBoxes()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>121</x>
     <y>73</y>
    </hint>
    <hint type="destinationlabel">
     <x>276</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>badge2Label</sender>
   <signal>clicked()</signal>
   <receiver>QtConfig</receiver>
   <slot>updateBoxes()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>231</x>
     <y>73</y>
    </hint>
    <hint type="destinationlabel">
     <x>276</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>badge3Label</sender>
   <signal>clicked()</signal>
   <receiver>QtConfig</receiver>
   <slot>updateBoxes()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>341</x>
     <y>73</y>
    </hint>
    <hint type="destinationlabel">
     <x>276</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>saveToConfig()</slot>
  <slot>addBadge(QTreeWidgetItem*)</slot>
  <slot>deleteBadge1()</slot>
  <slot>deleteBadge2()</slot>
  <slot>deleteBadge3()</slot>
  <slot>updateBoxes()</slot>
  <slot>openManualBadges()</slot>
 </slots>
</ui>
